js/editor.js

```diff
@@ -416,6 +430,15 @@ class Sketch extends Component {
         }).then(() => {
             firebase.database().goOffline();
         });
+
```

Comment:
@fdb Please ignore these comments, I was just trying the `POST` request on my firebase.

Reply:
Could you remove them in an additional commit?

Reply:
Sure.